plugins/modules/aci_interface_policy_leaf_fc_policy_group.py

```diff
@@ -0,0 +1,367 @@
+
+
+def main():
+    argument_spec = aci_argument_spec()
+    argument_spec.update(aci_annotation_spec())
+    argument_spec.update(aci_owner_spec())
+    argument_spec.update(
+        # NOTE: Since this module needs to include both infra:FcAccPortGrp (for FC) and infra:FcAccBndlGrp (for FC PC):
+        # NOTE: The user(s) can make a choice between (port(FC), port_channel(FC PC))
+        lag_type=dict(type="str", required=True, aliases=["lag_type_name"], choices=["port", "port_channel"]),
+        policy_group=dict(type="str", aliases=["name", "policy_group_name"]),  # Not required for querying all objects
+        description=dict(type="str", aliases=["descr"]),
+        fibre_channel_interface_policy=dict(type="str", aliases=["fibre_channel_interface_policy_name"]),
+        port_channel_policy=dict(type="str", aliases=["port_channel_policy_name"]),
+        attached_entity_profile=dict(type="str", aliases=["aep_name", "aep"]),
+        state=dict(type="str", default="present", choices=["absent", "present", "query"]),
+        name_alias=dict(type="str"),
+    )
+
+    module = AnsibleModule(
+        argument_spec=argument_spec,
+        supports_check_mode=True,
+        required_if=[
+            ["state", "absent", ["policy_group"]],
+            ["state", "present", ["policy_group"]],
+        ],
+    )
+
+    policy_group = module.params.get("policy_group")
+    description = module.params.get("description")
+    lag_type = module.params.get("lag_type")
+    fibre_channel_interface_policy = module.params.get("fibre_channel_interface_policy")
+    port_channel_policy = module.params.get("port_channel_policy")
+    attached_entity_profile = module.params.get("attached_entity_profile")
+    state = module.params.get("state")
+    name_alias = module.params.get("name_alias")
+
+    aci = ACIModule(module)
+
+    if lag_type == "port":
+        aci_class_name = "infraFcAccPortGrp"
+        dn_name = "fcaccportgrp"
+    elif lag_type == "port_channel":
+        aci_class_name = "infraFcAccBndlGrp"
+        dn_name = "fcaccbundle"
+
+    class_config_dict = dict(
+        name=policy_group,
+        descr=description,
+        nameAlias=name_alias,
+    )
+
+    child_configs = []
+    if fibre_channel_interface_policy is not None:
+        child_configs.append(
+            dict(
+                infraRsFcL2IfPol=dict(
+                    attributes=dict(
+                        tnFcIfPolName=fibre_channel_interface_policy,
+                    ),
+                ),
+            )
+        )
+    if attached_entity_profile is not None:
+        child_configs.append(
+            dict(
+                infraRsFcAttEntP=dict(
+                    attributes=dict(
+                        tDn="uni/infra/attentp-{0}".format(attached_entity_profile),
+                    ),
+                ),
+            )
+        )
+
+
```

Comment:
Sanity failed because of the additional line break so just remove it.

```suggestion
```

Reply:
resolved